indico/types/workflow.py

```diff
@@ -44,28 +46,41 @@ class Workflow(BaseType):
     component_links: List[WorkflowComponentLinks]
     created_by: int
     created_at: datetime
-    
+
     def component_by_type(self, component_type: str) -> WorkflowComponent:
         """
         Returns first component available of type specified.
         """
-        return next(component for component in self.components if component.component_type == component_type)
+        return next(
+            component
+            for component in self.components
+            if component.component_type == component_type
+        )
 
     def model_group_by_name(self, name: str) -> WorkflowComponent:
         """
         Returns first model group component of name specified.
         """
-        return next(component for component in self.components if hasattr(component, "model_group")
-                    and component.model_group.name == name)
+        return next(
+            component
+            for component in self.components
+            if hasattr(component, "model_group") and component.model_group.name == name
+        )
 
 
 class LinkedLabelStrategy(Enum):
-    BY_ROW = 0,
```

Comment:
we don't want to change this line-- should still be `BY_ROW = 0`

Reply:
Not sure how that happened, reverted back to old version